# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/kiax/kiax-iaxwrapper_includes.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN kiax-0.8.4/src/iaxwrapper.cpp kiax-0.8.4-new/src/iaxwrapper.cpp
--- kiax-0.8.4/src/iaxwrapper.cpp	2005-04-11 12:36:24.000000000 +0200
+++ kiax-0.8.4-new/src/iaxwrapper.cpp	2006-01-16 14:37:21.000000000 +0100
@@ -18,6 +18,8 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 #include "iaxwrapper.h"
+#include "malloc.h"
+#include "stdlib.h"
 
 IaxWrapper* callbackCaller;
 
